cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -737,8 +739,8 @@ function translateSVGPos(svgCanvas, clientX, clientY) {
     pt = pt.matrixTransform(svgCanvas.getScreenCTM().inverse());
 
     let pos = {
-        x: pt.x,
-        y: pt.y
+        x: pt.x - PLAYER_FRAME_OFFSET,
```

Comment:
Need to have corresponding methods to calculate coordinates from/to different coordinate systems.

Reply:
Done